grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -499,14 +501,17 @@ class NotifyService implements SendgridSupport, Status {
      * @param arguments NotifyArguments
      * @return Response is a map entry with true/false and a reason for failure, if failed.
      */
-    Map<Boolean, String> sendConsentGroupSecurityInfo(Issue issue, User user) {
+    Map<Boolean, String> sendRequirementsInfo(Issue issue, User user, String type) {
         Map<String, String> values = new HashMap<>()
         Map<Boolean, String> result = new HashMap<>()
 
         if (Boolean.valueOf(issue.getMTA())) {
             values.put(IssueExtraProperty.REQUIRE_MTA, "true")
         }
-        if (issue.getFeeForService() != null && Boolean.valueOf(issue.getFeeForService())) {
+        if (type != 'Project' && issue.getFeeForService() != null && Boolean.valueOf(issue.getFeeForService())) {
```

Comment:
`type` here really should be an enum. I try not to do raw string comparisons. I don't think we have an enum that exactly matches what you want here, but it seems to me that the best place for this would be `IssueType`.

Reply:
Could we solve this using a constant inside the class? IssueType is used to filter documents by type of issue. If I add another value the logic that uses IssueType it will try to filter by this new value.

Reply:
Yes, that could also work. Yes, `IssueType` might not be the best place for this, but it's where I would *expect* something like `Project` type.